### PR TITLE
Added all_options property into BootstrapSelect

### PR DESCRIPTION
**Pull request by quarckster**
This needs for PR https://github.com/ManageIQ/integration_tests/pull/4146

### CODE REVIEW

**Comment by mfalesni** [comment]
@quarckster https://travis-ci.org/RedHatQE/widgetastic.patternfly/jobs/225183974#L276

File: src/widgetastic_patternfly/__init__.py [hunk 1]
```diff
@@ -695,6 +695,15 @@ def all_selected_options(self):
                 parent=self)]
 
     @property
+    def all_available_options(self):
+        return [
+            {"text": self.browser.text(self.browser.element(
+                './/span[contains(@class, "text")]', parent=e)),
+            "value": e.get_attribute("data-original-index")}
+            for e in self.browser.elements('./div/ul/li', parent=self)
+        ]
```
Comment (mfalesni) [inline src/widgetastic_patternfly/__init__.py]: Does this have the same interface as original select?

Comment (quarckster) [inline src/widgetastic_patternfly/__init__.py]: No, it returns a dict not a named tuple

Comment (quarckster) [inline src/widgetastic_patternfly/__init__.py]: Now it has

**Pull request merged**